tests/test_space_ops.py

```diff
@@ -0,0 +1,10 @@
+import numpy as np
+
+from manim import cartesian_to_spherical, spherical_to_cartesian
+
+
+def test_polar_coords():
+    a = np.array([1, 1, 0])
+    b = (2, np.pi / 2, np.pi / 2)
+    assert cartesian_to_spherical(a) == (2 ** 0.5, np.pi / 2, np.pi / 4)
+    assert all(np.round(spherical_to_cartesian(*b)) == np.array([0, 2, 0]))
```

Comment:
Numpy's round method appears to default to 0 decimal places, which isn't really enough precision. Can you set it to something like 3 or 4?